taxi/forms.py

```diff
@@ -0,0 +1,56 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django import forms
+
+from taxi.models import Driver, Car
+
+
+def validate_license_number(lic_number: str) -> str:
+    if len(lic_number) != 8:
+        raise ValidationError("License number must be equal 8 symbols")
+    elif len(lic_number) == 8:
+        for i in range(0, 3):
+            if lic_number[i] != lic_number[i].capitalize():
+                raise ValidationError("First 3 letter must be capitalize")
```

Comment:
The error message should be more grammatically correct. Consider changing it to: 'The first 3 letters must be capitalized.'